src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -133,27 +133,31 @@ impl LogicalAgg {
                 TableCatalogBuilder::new(self.ctx().inner().with_options.internal_table_subset());
             let mut column_mapping = vec![];
 
+            let mut upstream_idx_to_state_idx = BTreeSet::new();
```

Comment:
I think the variable name should be changed accordingly😇. Maybe even no need to `mut` and `insert`, just `let group_keys = BTreeSet::from_iter(self.group_key().iter().cloned())` is ok.

Reply:
> I think the variable name should be changed accordingly😇. Maybe even no need to `mut` and `insert`, just `let group_keys = BTreeSet::from_iter(self.group_key().iter().cloned())` is ok.

fixed